src/redis.ts

```diff
@@ -0,0 +1,1256 @@
+    return this._commandParser(command, collection, destination, ...keys);
+  }
+  // End of SET commands
+
+  // Start of SORTED SET commands
+  zadd(key: string, data: any[], collection: string, options: any[] = []) {
+    const command: string = "ZADD";
+    return this._commandParser(command, collection, key, ...options, ...data);
+  }
+
+  zcard(key: string, collection: string) {
+    const command: string = "ZCARD";
+    return this._commandParser(command, collection, key);
+  }
+
+  zcount(key: string, minimum: string, maximum: string, collection: string) {
+    const command: string = "ZCOUNT";
+    return this._commandParser(command, collection, key, minimum, maximum);
+  }
+
+  zdiff(
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    withScores: boolean = false
+  ) {
+    const command: string = "ZDIFF";
+    let withScoresCommand: string | undefined;
+    if (withScores) {
+      withScoresCommand = "WITHSCORES";
+    } else {
+      withScoresCommand = undefined;
+    }
+    return this._commandParser(
+      command,
+      collection,
+      numKeys,
+      ...keys,
+      withScoresCommand
+    );
+  }
+
+  zdiffstore(
+    destination: string,
+    numkeys: number,
+    keys: string[],
+    collection: string
+  ) {
+    const command: string = "ZDIFFSTORE";
+    return this._commandParser(
+      command,
+      collection,
+      destination,
+      numkeys,
+      ...keys
+    );
+  }
+
+  zincrby(key: string, increment: number, member: string, collection: string) {
+    const command: string = "ZINCRBY";
+    return this._commandParser(command, collection, key, increment, member);
+  }
+
+  zinter(
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    options: any[] | undefined = undefined,
+    withScores: boolean = false
+  ) {
+    const command: string = "ZINTER";
+
+    let optionsArray: any[] = [];
+    if (options !== undefined) {
+      optionsArray = [...options];
+    }
+    if (withScores) {
+      optionsArray.push("WITHSCORES");
+    }
+
+    return this._commandParser(
+      command,
+      collection,
+      numKeys,
+      ...keys,
+      ...optionsArray
+    );
+  }
+
+  zinterstore(
+    destination: string,
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    options: any[] = []
+  ) {
+    const command: string = "ZINTERSTORE";
+    return this._commandParser(
+      command,
+      collection,
+      destination,
+      numKeys,
+      ...keys,
+      ...options
+    );
+  }
+
+  zlexcount(key: string, min: string, max: string, collection: string) {
+    const command: string = "ZLEXCOUNT";
+    return this._commandParser(command, collection, key, min, max);
+  }
+
+  zmscore(key: string, members: string[], collection: string) {
+    const command: string = "ZMSCORE";
+    return this._commandParser(command, collection, key, ...members);
+  }
+
+  zpopmax(
+    key: string,
+    collection: string,
+    count: number | undefined = undefined
```

Comment:
make it optional if it could be undefined.

Reply:
👍  done